docs/advanced-usage/access-control/kerberos/index.md

```diff
@@ -47,10 +47,22 @@ $ reana-client secrets-add --env CERN_USER=johndoe \
 
 ## Setting Kerberos requirement
 
-Set `kerberos: true` for the steps in need in the workflow specification.
-Please note that step's docker image (e.g `environment: 'cern/slc6-base'`)
-should have Kerberos client installed and you have to for the Kerberos
-authentication to work.
+If the workflow engine you are using needs Kerberos to parse and validate the
+workflow specification, then you can enable it globally for the whole workflow
+in the `reana.yaml` file. For example, this is needed if you want to access
+restricted resources directly from your `Snakefile`. This will also set up
+Kerberos for each step of your workflow.
+
+```yaml hl_lines="4"
+workflow:
+  type: snakemake
+  resources:
+    kerberos: true
+  file: workflow/snakemake/Snakefile
+```
+
```

Comment:
This will enable Kerberos authentication not only for workflow orchestration, but also for each workflow step job.